libraries/NetworkClientSecure/src/NetworkClientSecure.cpp

```diff
@@ -305,9 +305,11 @@ int NetworkClientSecure::available() {
   res = data_to_read(sslclient.get());
 
   if (res < 0 && !_stillinPlainStart) {
-    log_e("Closing connection on failed available check");
+    if (res != -30848) {
```

Comment:
what is `-30848`? It should be an error name

Reply:
it  in ssl_client.cpp _handle_error too. I didn't find that number in any header file

Reply:
now I found this https://github.com/Mbed-TLS/mbedtls/issues/2173

Reply:
and then this https://github.com/espressif/arduino-esp32/issues/9230